### PR TITLE
integrated with backend

### PR DESCRIPTION
**Pull request by anantKC**

### CODE REVIEW

File: src/modules/login/views/ViewResetPassword.vue [hunk 1]
```diff
@@ -1,172 +1,182 @@
...
-          submission:{
-            message:'',
-            isVerified:false
-          }
-          
-        };
+import BaseInput from "@/components/BaseInput.vue";
+import validatePassword from "@/utils/validatePassword.js";
+import { resetPassword } from "@/modules/register/services/reset_index.js";
+import BaseAlert from "../../../components/BaseAlert.vue";
```
Comment (asimnp) [inline src/modules/login/views/ViewResetPassword.vue]: Use "@" instead of "../../../"

File: src/modules/login/views/ViewResetPassword.vue [hunk 2]
```diff
@@ -1,172 +1,182 @@
...
-        },
-        formData(){
-          return{
-            data:{
-            password:this.password.value,
-            },
-            token:"1bxXVTgMmdgiClqXZ8Rdmg"
-          }
+    toggleSeen(field) {
+      if (field == "cpassword") {
```
Comment (asimnp) [inline src/modules/login/views/ViewResetPassword.vue]: Use "===" strict checking

File: src/modules/login/views/ViewResetPassword.vue [hunk 3]
```diff
@@ -1,172 +1,182 @@
...
+        this.cpassword.seen = !this.cpassword.seen;
+      }
+      if (field == "password") {
```
Comment (asimnp) [inline src/modules/login/views/ViewResetPassword.vue]: Use "===" strict checking

File: src/modules/login/views/ViewResetPassword.vue [hunk 4]
```diff
@@ -1,172 +1,182 @@
...
+        return;
+      }
+      if (!this.password.error && !this.cpassword.error) {
+        try {
+          const response = await resetPassword(this.formData());
+          this.submission = {
+            message: "Password Changed Successfully",
+            isVerified: true,
+          };
+          console.log("response:", response);
```
Comment (asimnp) [inline src/modules/login/views/ViewResetPassword.vue]: Remove console.log

File: src/modules/register/services/reset_index.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import apiUrl from '@/constants/routes/resetPassword'
+
+import API from "@/services/API"
+
+
+ const resetPassword = async (resetPassword)=>{
+    const config = {
+        headers:{
+            'Content-Type': 'application/json',
+            Authorization: 'Bearer ' + "1bxXVTgMmdgiClqXZ8Rdmg"
```
Comment (asimnp) [inline src/modules/register/services/reset_index.js:10]: Why the token is static?

Comment (anantKC) [reply src/modules/register/services/reset_index.js:10]: it will be made dynamic once it is merged

**Review by asimnp — APPROVED** [review]

**Pull request merged**